src/js/mailchimp.js

```diff
@@ -0,0 +1,33 @@
+const setupEmailSignupForm = () => {
+  const $form = $(".newsletter-form")
+  $(".newsletter-form .signup-link").click(event => {
+    event.preventDefault()
+    $form.submit()
+  })
+
+  $form.submit(async event => {
+    event.preventDefault()
+    const url = $form.attr("action")
+    for (const item of $form.serializeArray()) {
+      if (item.name === "EMAIL" && !item.value) {
+        // ignore for empty emails
+        return
+      }
+    }
+
+    const response = await $.ajax({
+      url:      url,
+      data:     $form.serialize(),
+      dataType: "jsonp",
+      method:   $form.attr("method")
+    })
+    const { result, msg } = response
+    if (result === "error") {
+      $form.html(msg)
```

Comment:
This replaces the input with the error message, preventing the user from correcting a mistake.  Can it be appended underneath the input instead?

<img width="616" alt="Screen Shot 2020-11-05 at 3 43 02 PM" src="https://user-images.githubusercontent.com/187676/98294312-c0be4480-1f7d-11eb-8b50-874c5b739af1.png">